doc/en/api/1.0.0/gwcfilterupdate.yaml

```diff
@@ -0,0 +1,47 @@
+---
+swagger: '2.0'
+info:
+  version: 1.0.0
+  title: GeoWebCache Filter Update
+  description: The REST API for Updating GWC filter
+schemes:
+- http
+host: localhost:8080
+basePath: /geoserver/gwc/rest
+paths:
+  /filter/{filterName}/update/{updateType}:
+    post:
+      consumes:
+        - application/xml
+        - application/zip
+      operationId: filterUpdatePost
+      summary: Updates a given layer filter by way of xml or zip file.
+      description: Restfully updates the given filterName with parameters provided in the xml or zip
+      parameters:
+        - name: filterName
```

Comment:
request body should be one of the parameters

Reply:
Seeking clarification on this, I'm now not sure if I'm using the correct parameters for this endpoint - could be one of:
http://geowebcache.org/docs/1.11.0/configuration/layers/parameterfilters.html
or
http://geowebcache.org/docs/1.11.0/configuration/layers/requestfilters.html

The latter has limited documentation and neither has much documentation for REST endpoints.

Reply:
I've left the request body out of this endpoint, it is looking for the requestfilters - perhaps @smithkm can help me out with this one at some point.

Reply:
Looking at [FilterUpdateController](https://github.com/GeoWebCache/geowebcache/blob/master/geowebcache/rest/src/main/java/org/geowebcache/rest/controller/FilterUpdateController.java#L83), I believe it is dealing with RequestFilters.

As for the structure of the body, I can't find anything useful; I agree we probably need help from @smithkm